packages/ckeditor5-image/ckeditor5-metadata.json

```diff
@@ -7,7 +7,70 @@
 			"docs": "features/image.html#inserting-images-via-pasting-url-into-editor",
 			"path": "src/autoimage.js",
 			"requires": [
-				"Image"
+				[
+					"ImageBlock",
+					"ImageInline"
+				]
+			]
+		},
+		{
+			"name": "Block image",
+			"className": "ImageBlock",
+			"description": "Enables support for inserting images between the blocks of text, like paragraphs or block quotes. This functionality is also provided by the main Image plugin. Note: The image feature is introduced in a granular form, implemented by separate plugins responsible for such aspects as image caption, toolbar, resize, upload or styles.",
+			"path": "src/imageblock.js",
+			"uiComponents": [
+				{
+					"type": "Button",
+					"name": "imageTextAlternative",
+					"iconPath": "",
+					"toolbars": [
+						"image.toolbar"
+					]
+				}
+			],
+			"htmlOutput": [
+				{
+					"elements": "figure",
+					"classes": "image"
+				},
+				{
+					"elements": "img",
+					"attributes": [
+						"alt",
+						"sizes",
+						"src",
+						"srcset",
+						"width"
+					]
+				}
+			]
```

Comment:
Since the new `ImageBlock` and `ImageInline` plugins are the ones that create HTML elements (`<figure>`, `<img>`) now, then I wonder if the "old" `Image` plugin should still have the `htmlOutput` and `uiComponents` arrays here, as it does not create those things anymore.

Reply:
I tried to find a similar case where there's a glue plugin and individual features underneath. I think it mainly depends on how this data will be consumed.

Reply:
> I tried to find a similar case where there's a glue plugin and individual features underneath. I think it mainly depends on how this data will be consumed.

It can be consumed either by some tools like Online Builder, either by developers.

For me, we could try to generate the `Image`'s `htmlOutput` and `uiComponents` dynamically (in the OB and HTML output guide) as we know that it consists of `ImageBlock` and `ImageInline`. So I'd leave these fields empty as we can generate them dynamically (of course this behavior would have to be documented).